compiler/src/main/kotlin/ast/ast.kt

```diff
@@ -0,0 +1,147 @@
+package dk.aau.cs.d409f19.cellumata.ast
+
+import org.antlr.v4.runtime.tree.ParseTree
+
+/**
+ * This is a debugging string.
+ * If you see this string in running code it means that the variable hasn't been initialized.
+  */
+const val MAGIC_UNDEFINED_STRING = "<<THIS IS A MAGIC STRING, UNDEFINED>>"
+
+sealed class AST
+
+data class RootNode(val world: WorldNode, val body: List<Decl>) : AST()
+
+/*
+* World definition
```

Comment:
We should distributed a codestyle-config for IntelliJ, which the group agrees upon.

Reply:
I agree